openlibrary/plugins/openlibrary/js/ol.js

```diff
@@ -255,7 +258,7 @@ $(function(){
         };
     };
 
-    $(document).on('submit','.trigger', function(e) {
```

Comment:
this line shouldn't have been changed, cc: @ShreyashApp this was the cause of the regression to your fix :)

Reply:
That was done by mistake. Thanks @mekarpeles  for pointing it out for me.